packages/generators/src/init-generator.ts

```diff
@@ -201,7 +201,24 @@ export default class InitGenerator extends CustomGenerator {
                 use: ExtractUseProps,
             });
         }
-        if (this.usingDefaults) {
+
+        // webpack Dev Server
+        const { useDevServer } = await Confirm(self, 'useDevServer', 'Do you want to use webpack-dev-server', true, false);
+        if (useDevServer) {
+            this.dependencies.push('webpack-dev-server');
+            this.configuration.config.webpackOptions.devServer = {
+                open: true,
```

Comment:
Let's add host: `localhost` too, just for example

Reply:
yep lets do it, I think it's fine to make `localhost` as default